src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEmployeeInfoValidationServiceImpl.java

```diff
@@ -0,0 +1,119 @@
+    protected PayeeACHService payeeACHService;
+    protected ConfigurationService configurationService;
+
+    @Override
+    public boolean validPerson(String employeeId) {
+        Person employee = findPerson(employeeId);
+        return validPerson(employee);
+    }
+
+    private boolean validPerson(Person employee) {
+        boolean valid = ObjectUtils.isNotNull(employee);
+        if (LOG.isDebugEnabled()) {
+            if (valid) {
+                LOG.debug("validPerson, the employee id " + employee.getEmployeeId() + " was built into the person: " + employee.getName());
+            }
+        }
+        return valid;
+    }
+    
+    private Person findPerson(String employeeId) {
+        if (StringUtils.isNotBlank(employeeId)) {
+            try {
+                Person employee = getPersonService().getPersonByEmployeeId(employeeId);
+                return employee;
+            } catch (Exception e) {
+                LOG.error("findPerson, Unable to retrieve a person from employee ID: " + employeeId, e);
+            }
+        } else {
+            LOG.error("findPerson, No employee ID was provided");
+        }
+        return null;
+    }
+
+    @Override
+    public boolean validPdpAddress(String employeeId) {
+        Person employee = findPerson(employeeId);
+        boolean validPerson = validPerson(employee);
+        boolean validAddress = validPerson;
+        if (validPerson) {
+            String state = employee.getAddressStateProvinceCodeUnmasked();
+            String country = employee.getAddressCountryCodeUnmasked();
+            if (StringUtils.isBlank(state) && StringUtils.isBlank(country)) {
+                validAddress = false;
+                LOG.error("validPdpAddress, " + employee.getName() + " does not have a Country Code or a State/Province code.");
+            } else {
+                if (LOG.isDebugEnabled()) {
+                    LOG.debug("validPdpAddress, The employee " + employee.getName() + " had a country code or state code." );
+                }
+            }
+        }
+        return validAddress;
+    }
+    
+    @Override
+    public boolean isPayeeSignedUpForACH(String employeeId) {
+        boolean isACH = getPayeeACHService().isPayeeSignedUpForACH(ConcurConstants.EMPLOYEE_PAYEE_STATUS_TYPE_CODE, employeeId);
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("isPayeeSignedUpForACH, is employee ID " + employeeId + " signed up for ACH: " + isACH );
+        }
+        return isACH;
+    }
+    
+    @Override
+    public String getAddressValidationMessageIfCheckPayment(String employeeId) {
+        String validationMessage = StringUtils.EMPTY;
+        if (isPayeeSignedUpForACH(employeeId)) {
+            LOG.info("validateAddressIfCheckPayment, the employee ID " + employeeId + " is signed up for ACH so no need to validdate address.");
+        } else {
+            boolean valid = validPdpAddress(employeeId);
+            if (!valid) {
```

Comment:
you could just check validPdpAddress(employeeId) here since the valid variable is not used for anything else